src/test/java/database/SaveLoadPlayerTest.java

```diff
@@ -0,0 +1,141 @@
+        FileWriter file = new FileWriter("src/main/java/database/players.csv");
+        file.write("");
+        file.close();
+    }
+
+    /**
+     * Saves and loads one valid player (player exists in the file).
+     * @throws IOException If an input/output error occurs when writing to the file.
+     */
+    @Test
+    public void saveLoadOneValidPlayer() throws IOException {
+        database.savePlayer(p1);
+        PlayerInformation p4 = database.loadPlayer("Lily1");
+
+        assert p4 != null;
+        assertEquals(p1.getName(), p4.getName());
+        assertEquals(p1.getWins(), p4.getWins());
+        assertEquals(p1.getLosses(), p4.getLosses());
+    }
+
+    /**
+     * Saves and loads multiple valid players (players exist in the file).
+     * @throws IOException If an input/output error occurs when writing to the file.
+     */
+    @Test
+    public void saveLoadMultipleValidPlayer() throws IOException {
+        database.savePlayer(p1);
+        database.savePlayer(p2);
+        PlayerInformation p4 = database.loadPlayer("Lily1");
+        PlayerInformation p5 = database.loadPlayer("Lily2");
+
+        assert p4 != null;
+        assertEquals(p1.getName(), p4.getName());
+        assertEquals(p1.getWins(), p4.getWins());
+        assertEquals(p1.getLosses(), p4.getLosses());
+
+        assert p5 != null;
+        assertEquals(p2.getName(), p5.getName());
+        assertEquals(p2.getWins(), p5.getWins());
+        assertEquals(p2.getLosses(), p5.getLosses());
+    }
+
+    /**
+     * Loads one invalid player (player does not exist in the file).
+     * @throws IOException If an input/output error occurs when writing to the file.
+     */
+    @Test
+    public void loadOneInvalidPlayer() throws IOException {
+        PlayerInformation p4 = database.loadPlayer("Lily3");
+
+        assertNull(p4);
+    }
+
+    /**
+     * Loads multiple invalid players (players do not exist in the file).
+     * @throws IOException If an input/output error occurs when writing to the file.
+     */
+    @Test
+    public void loadMultipleInvalidPlayer() throws IOException {
+        PlayerInformation p4 = database.loadPlayer("Lily3");
+        PlayerInformation p5 = database.loadPlayer("Lily4");
+
+        assertNull(p4);
+        assertNull(p5);
+    }
+
+    /**
+     * Saves and loads one valid player, loads one invalid player.
+     * @throws IOException If an input/output error occurs when writing to the file.
+     */
+    @Test
+    public void saveLoadValidInvalidPlayer() throws IOException {
+        database.savePlayer(p1);
+        PlayerInformation p4 = database.loadPlayer("Lily1");
+        PlayerInformation p5 = database.loadPlayer("Lily3");
+
+        assert p4 != null;
+        assertEquals(p1.getName(), p4.getName());
+        assertEquals(p1.getWins(), p4.getWins());
+        assertEquals(p1.getLosses(), p4.getLosses());
+
+        assertNull(p5);
+    }
+
+    /**
+     * Saves an existing player (should not save).
```

Comment:
Is this supposed to save or not save the player? The wording is a bit confusing here.